### ISSUE TITLE
Core functionality

### ISSUE THREAD

**Issue by bmind12** [event-maker-telegram-bot/event-maker-telegram-bot#13]
Implement the core functionality of the bot.

**What should be implemented:**

The bot should react on the `New event` command and should consequently ask a user for:
1. an event name (without any validation)
1. the event date (providing a guide on how to format it) (asking until the correct input format is received)
1. is it a full-day event? (_yes_ or _no_) If _no_ (`false`), the bot asks for:
    1. the event start time (providing a guide on how to format it) (asking until the correct input format is received)
    1. the event end time (providing a guide on how to format it) (asking until the correct input format is received)
1. the event location (string value without validation is enough) or _no_ for empty location

**What should NOT be implemented (but will be in another issue):**

1. event (file and links) generation
1. additional validation where needed, e.g. validate if dates make sense (start date is before end date)
**Dev notes:**

- This depends on #11, the command name should be aligned with the one which is used when customising the bot with Botfather.
- During implementation we should consider the best way to temporary store the received data, until all the answers are received.
- See more info on [MVP](https://github.com/bmind12/event-maker-telegram-bot/wiki/MVP#mvp-specs) wiki page

**Comment by fidelman** [comment]
> What should NOT be implemented (but will be in another issue)

Will be covered in #23 and #22

**Comment by fidelman** [comment]
Considering #23, I think the steps should be done a declarative way in a `.json` file
